libevm/VMFactory.cpp

```diff
@@ -137,10 +166,10 @@ po::options_description vmProgramOptions(unsigned _lineLength)
     auto add = opts.add_options();
 
     add("vm",
-        po::value<VMKind>()
-            ->value_name("<name>")
-            ->default_value(VMKind::Legacy, "legacy")
-            ->notifier(VMFactory::setKind),
+        po::value<std::string>()
+            ->value_name("<name>|<path>")
+            ->default_value("legacy")
+            ->notifier(setVMKind),
```

Comment:
Remove `VMFactory::setKind` f it's not used anymore

Reply:
This is public API, but not used now. I will remove it because not thread-safe.

Reply:
Removed.